synapse/lib/types.py

```diff
@@ -637,7 +646,15 @@ def postTypeInit(self):
             raise s_exc.BadConfValu(name='size', valu=self._size,
                                     mesg='Size must be a multiple of 2')
 
-        self._zeropad = self.opts.get('zeropad')
+        if self._zeropad < 0:
+            raise s_exc.BadConfValu(name='zeropad', valu=self._zeropad,
+                                    mesg='Zeropad must be > 0')
```

Comment:
must be >= 0

Reply:
Good catch. I copied the size error block from above which has the same thing. Bit by copypasta.